### PR TITLE
Added Tests for UserManagerService

### PR DESCRIPTION
**Pull request by Vinnytile**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by danylo-melnychuk — APPROVED** [review]
Welldone

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockMapper = new Mock<IMapper>();
+            var store = new Mock<IUserStore<User>>();
+            _mockUserManager = new Mock<UserManager<User>>(store.Object, null, null, null, null, null, null, null, null);
+            _userManagerService = new UserManagerService(_mockUserManager.Object, _mockMapper.Object);
+        }
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:91]: can you move this section to the beginning?

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+namespace EPlast.Tests.Services.UserManager
+{
+    class UserManagerServiceTests
+    {
+        private UserManagerService _userManagerService;
+        private Mock<UserManager<User>> _mockUserManager;
+        private Mock<IMapper> _mockMapper;
+
+        private readonly List<string> _roles = new List<string>();
+        private readonly UserDTO _userDTO = new UserDTO();
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:20]: Why don't you use SetUp for this?

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+using EPlast.DataAccess.Entities;
+using Microsoft.AspNetCore.Identity;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace EPlast.Tests.Services.UserManager
+{
+    class UserManagerServiceTests
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:13]:
```suggestion
    [TestFixture]
    public class UserManagerServiceTests
```

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                .Setup(x => x.IsInRoleAsync(It.IsAny<User>(), It.IsAny<string>()))
+                .ReturnsAsync(false);
+            
+            //Act
+            var result = await _userManagerService.IsInRoleAsync(_userDTO, roles);
+
+            //Assert
+            Assert.IsFalse(result);
+        }
+        [Test]
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:53]: add space

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+using System.Threading.Tasks;
+
+namespace EPlast.Tests.Services.UserManager
+{
+    class UserManagerServiceTests
+    {
+        private UserManagerService _userManagerService;
+        private Mock<UserManager<User>> _mockUserManager;
+        private Mock<IMapper> _mockMapper;
+
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:18]: ?

**Review by voroniak — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs (hunk 5 above)
Comment (Vinnytile) [reply EPlast/EPlast.Tests/Services/UserManager/UserManagerServiceTests.cs:18]: What's wrong with this part?

**Review by tanyabiryuk — APPROVED** [review]

**Review by Oleh-Bodnar — APPROVED** [review]

**Review by voroniak — APPROVED** [review]

**Pull request merged**